### DISCUSSION TITLE
How to override django local Installed app in saleor platform

### DISCUSSION THREAD

**Discussion by jaswant99k** [saleor/saleor-platform#234] [General]
I have to make changes (override) some codes inside the locally installed django  app.

what are the best practices or how can I do without disturbing existing code?

Thanks

**Comment by peelar** [comment]
Hello @jaswant99k 👋 

I am not sure what your use case is, but I see two options:
1) Forking the repository.
2) Creating an app that extends Saleor. You can build it with any technology you like. [Our documentation](https://docs.saleor.io/docs/3.x/developer/extending/apps/key-concepts) focuses on JavaScript, but there is also a [Python App Framework](https://github.com/mirumee/saleor-app-framework-python) we can recommend.
